internal/server/iigointernal/executive.go

```diff
@@ -73,29 +81,50 @@ func (e *executive) broadcastTaxation(islandsResources map[shared.ClientID]share
 			communicateWithIslands(islandID, shared.TeamIDs[e.ID], data)
 		}
 	}
+	return nil
```

Comment:
Service charge without islands decision to take the action, same as below in replyAllocationRequest